### ISSUE TITLE
NFC_SETUP: Didn't find PN53x board - Lost connection after some minutes / hours

### ISSUE THREAD

**Issue by RedJohn14** [rednblkx/HomeKey-ESP32#125]
Hello!

I used some ESP32-S and wire everything correct. NFC reader LED is on. But everytime I started the ESP get this error message:

NFC_SETUP: Didn't find PN53x board

Any ideas what my mistake is? What i can do to run this setup correctly :-(

See also my settings:
![279D090D-5856-4AA7-A995-794A55C882BB](https://github.com/user-attachments/assets/b2a5e435-ddc4-4eb4-9e41-55764dcaa73d)
![710A571A-98A8-4107-9C27-F352764CB8EC_4_5005_c](https://github.com/user-attachments/assets/ccefcd76-b94e-48fe-b4f6-dcca29daf2e9)
![AF70F670-433E-4321-AB71-5AA3221D0783](https://github.com/user-attachments/assets/ce63995c-87e5-4c36-ab75-8192fc817e27)

**Comment by meluvalli** [comment]
Did you see issue #124?

**Comment by RedJohn14** [comment]
> Did you see issue #124?

Yes I see this post and buy "the not china" version of the nfc board. Now it works but only for some minutes or hours (max 1 hours.) after this time i must restart the esp32 and with a littel bit luck it will work than. So the problem is that the connection to the nfc ist lost but led on nfc board is still on. any ideas?

Thanks!

**Comment by rednblkx** [comment]
Please try the latest version, I’ve lowered the spi clock speed, It is possible the PN532 cannot remain stable at that speed for some reason. Fake ICs are also unfortunately a thing.
